### PR TITLE
Update Chromium data for webextensions.api.storage.StorageArea

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates and corrects version values for Chromium (Chrome, Opera, Samsung Internet, WebView Android) for the `StorageArea` member of the `storage` Web Extensions interface. This sets the feature(s) to a version range based upon the date that the feature was added to BCD with the intent of replacing `true` values with ranged values to eliminate `true` values from BCD.

Commit/PR Adding the Feature: #17826

### CODE REVIEW

File: webextensions/api/storage.json [hunk 1]
```diff
@@ -118,7 +118,7 @@
                 "description": "Supports empty key",
                 "support": {
                   "chrome": {
-                    "version_added": true
+                    "version_added": "≤93"
```
Comment (rebloor) [inline webextensions/api/storage.json:121]: @queengooborg the referenced PR that added these were dated Sep 20, 2022. Therefore, shouldn't this be version 105?

**Review by rebloor — APPROVED** [review]

File: webextensions/api/storage.json (hunk 1 above)
Comment (queengooborg) [reply webextensions/api/storage.json]: Uh oh -- 105 is just barely too early to satisfy the linter...

Comment (rebloor) [reply webextensions/api/storage.json]: @dotproto any idea on how to find out when CHrome started support for empty kets in storage.get?

Comment (queengooborg) [reply webextensions/api/storage.json]: Honestly?  You would have a better idea about how to track that down than I would...

Chrome 105 is _very_ close to being within range, though -- in fact, it's just one version away.  Perhaps we could set this to 104 and call it good?  (The other alternative is to wait until September, once 105 is older than two years?)

**Pull request merged**